integration/resources/ddo-nft.json

```diff
@@ -328,6 +328,11 @@
                   "name": "_numberNfts",
                   "type": "uint256",
                   "value": "1"
+                },
+                {
+                  "name": "_contractAddress",
+                  "type": "address",
+                  "value": "1"
```

Comment:
this should instead be done using `yarn utils:generate-ddos` to make sure the sdk is building the ddos correctly